shared/middleware/helpers/storage_apps.rb

```diff
@@ -14,7 +14,7 @@ def initialize(storage_id)
   def create(value, ip:, type: nil, published_at: nil)
     timestamp = DateTime.now
     row = {
-      storage_id: @storage_id,
+      storage_id: @storage_id.to_i,
```

Comment:
Might need your help, as the failing tests pass fine locally, making it hard to debug. Any idea what might be causing the difference in environment behavior?